apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -28,10 +28,12 @@
 #include <signal.h>
 #include <time.h>
 #include <stdbool.h>
+#include <wait.h>
```

Comment:
There is no <wait.h>. Is it <sys/wait.h>?

Reply:
modify